cli/tests/pcluster/createami/test_pcluster_createami.py

```diff
@@ -100,12 +104,13 @@ def test_validate_createami_args_architecture_compatibility(
     )
     if error_expected:
         with pytest.raises(SystemExit) as sysexit:
-            createami._validate_createami_args_architecture_compatibility(args)
+            createami._validate_createami_args_ami_compatibility(args)
         assert_that(sysexit.value.code).is_not_equal_to(0)
     else:
-        assert_that(createami._validate_createami_args_architecture_compatibility(args)).is_equal_to(
-            base_ami_architecture
+        assert_that(createami._validate_createami_args_ami_compatibility(args)).is_equal_to(
+            {"Architecture": base_ami_architecture, "Name": base_ami_name}
         )
+        createami.utils.validate_pcluster_version_based_on_ami_name.assert_called_with(base_ami_name)
```

Comment:
Minor: I would prefer not to call the mocked function like this, because from this code alone we can't really tell if this function is mocked.

Alternatively, we can do the following:
```
version_validation_spy = mocker.patch("pcluster.createami.utils.validate_pcluster_version_based_on_ami_name")
...
version_validation_spy.assert_called_with(base_ami_name)
```

Reply:
Yeah. This is the first approach that came to my mind. I wrote the current code to keep a consistent style with the code surrounding